src/locale/locales/es/messages.po

```diff
@@ -1835,58 +1835,58 @@ msgstr "Dominio personalizado"
 #: src/view/screens/Feeds.tsx:760
 #: src/view/screens/Search/Explore.tsx:391
 msgid "Custom feeds built by the community bring you new experiences and help you find the content you love."
-msgstr ""
+msgstr "Los feeds personalizados creados por la comunidad te brindan nuevas experiencias y te ayudan a encontrar el contenido que te encanta."
 
 #: src/view/screens/PreferencesExternalEmbeds.tsx:54
 msgid "Customize media from external sites."
 msgstr "Preferencias sobre medios externos."
 
 #: src/components/dialogs/PostInteractionSettingsDialog.tsx:289
 msgid "Customize who can interact with this post."
-msgstr ""
+msgstr "Personaliza quién puede interactuar con esta publicación."
 
 #: src/screens/Settings/AppearanceSettings.tsx:109
 #: src/screens/Settings/AppearanceSettings.tsx:130
 msgid "Dark"
-msgstr ""
+msgstr "Oscuro"
 
 #: src/view/screens/Debug.tsx:63
 msgid "Dark mode"
-msgstr ""
+msgstr "Modo Oscuro"
 
 #: src/screens/Settings/AppearanceSettings.tsx:122
 msgid "Dark theme"
-msgstr ""
+msgstr "Tema oscuro"
 
 #: src/view/screens/Settings/index.tsx:473
 #~ msgid "Dark Theme"
 #~ msgstr ""
 
 #: src/screens/Signup/StepInfo/index.tsx:222
 msgid "Date of birth"
-msgstr ""
+msgstr "Fecha de nacimiento"
 
 #: src/screens/Settings/components/DeactivateAccountDialog.tsx:73
 #: src/view/screens/Settings/index.tsx:772
 msgid "Deactivate account"
-msgstr ""
+msgstr "Desactivar cuenta"
 
 #: src/view/screens/Settings/index.tsx:784
 msgid "Deactivate my account"
-msgstr ""
+msgstr "Desactivar mi cuenta"
 
 #: src/view/screens/Settings/index.tsx:839
 msgid "Debug Moderation"
-msgstr ""
+msgstr "Moderacion de debug"
 
 #: src/view/screens/Debug.tsx:83
 msgid "Debug panel"
-msgstr ""
+msgstr "Panel de debug"
```

Comment:
`debug` -> `depuración`

Same for almost all the other instances of this I'd say.

Reply:
Fixed all